lunatrace/bsl/frontend/src/pages/project/builds/details/vulnerable-packages/vulnerable-package-card/VulnerablePackageMain.tsx

```diff
@@ -113,11 +118,25 @@ export const VulnerablePackageMain: React.FunctionComponent<VulnerablePackageMai
     );
   };
 
+  async function onClickUpdate(pkg: VulnerablePackage) {
+    const response = await createGitHubPullRequestForVuln({
+      project_id: build.project_id,
+      vulnerability_id: pkg.affected_by[0].vulnerability.id,
+      old_package_slug: `${pkg.release.package.name}@${pkg.release.version}`,
+      new_package_slug: `${pkg.release.package.name}@^${recommendedVersion}`,
+      // TODO: Add support for multiple paths...
+      package_manifest_path: pkg.paths[0],
+    });
+
+    // TODO: Make this actually put some HTML on the page.
+    console.log('pr response:', response);
```

Comment:
we should probably finish this feature before merging to master. 

Id like to keep master in a deployable state at all times